### DISCUSSION TITLE
Selecting multiple episodes

### DISCUSSION THREAD

**Discussion by VishalTyagi11** [justchokingaround/lobster#37] [Q&A]
Is it possible to select multiple episodes as is possible in ani-cli?

**Comment by codinPenguin** [comment]
did you ever get an answer?

**Reply by VishalTyagi11** [reply]
Nope

**Reply by justchokingaround** [reply]
sorry for being unactive, im quite busy with uni work atm, this can be implemented, i'll try looking into it during the christmas holidays :+1:

**Reply by VishalTyagi11** [reply]
Sure, man. Keep up the good work!!!
